lib/measures/openstudio_results/tests/os_results.osw

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
should this file be in here? seem likes it is a result.

Reply:
@nllong It should stay it is an input for a test. https://github.com/NREL/openstudio-common-measures-gem/blob/3_1_0_upgrade/lib/measures/openstudio_results/tests/OpenStudioResults_Test.rb#L85